### ISSUE TITLE
Suggestion: Checking WAV files ONLY supports MONO channel.

### ISSUE THREAD

**Issue by jaigitrep159** [TheUnpunished/gintool#7]
# Why?

The reason for that is because when doing it in stereo, it glitched out. Even using stereo in 48000hz.

This suggestion here i said, when GINTool detects a WAV file if it's mono, then it is accepted, but if it checks stereo, then it will cause an error saying like: "This WAV file must be in MONO channel only." and if it checks the exported grain folder containing wav files had stereo in it, it will cause an error as well.

I tested it on 32bit float WAV File, 48000hz in mono channel, and REV actually converts it into 16 bit PCM, and GINTool converts a 32-bit float to 16-bit pcm using ffmpeg. Generate the GIN file and it worked in-game (Tested in NFSC 1.4 Collectors Edition), which is actually good for that.

it doesn't matter if it's 44100hz or 48000hz, it just works in mono like it should. But in stereo, it'll glitch out like pixels (mainly the file size is bigger, which tested on foobar2000 media player with vgnstream plugin).

# In conclusion...
Always make a WAV file, whether is 32-bit float or 16-bit PCM, has to be MONO, (exported grain folder as well) and if GINTool checks it, it will start the process, if it's STEREO, then it will not make a process, so it must supposed to be in MONO channel.

**Comment by TheUnpunished** [comment]
When you open .wav files in REV, you can already see whether the audio is stereo or mono.
I will add a check for it.
